scripts/build-ghp-index.js

```diff
@@ -0,0 +1,100 @@
+import fs from "fs";
+
+const tokenList = JSON.parse(
+  fs.readFileSync("src/hemi.tokenlist.json", "utf-8"),
+);
+
+const prefix =
+  "https://raw.githubusercontent.com/hemilabs/token-list/master/src/";
+
+const toRelative = (url) =>
+  url.startsWith(prefix) ? url.slice(prefix.length) : url;
+
+const logo = ({ logoURI, name }) => `
+<img alt="${name} token icon" class="inline min-w-6 size-6" src="${toRelative(logoURI)}" />
+`;
+
+const shortenAddress = (address) =>
+  `${address.slice(0, 6)}...${address.slice(-4)}`;
+
+const row = ({ name, chainId, symbol, address, logoURI }) => `
+<tr class="h-12 border-y">
+  <td class="px-4 py-3">
+    ${logo({ logoURI, name })}
+    <span class="hidden lg:inline overflow-hidden whitespace-nowrap ml-2 text-ellipsis">${name}</span>
+  </td>
+  <td class="overflow-hidden whitespace-nowrap px-4 py-3 text-ellipsis">${symbol}</td>
+  <td class="overflow-hidden whitespace-nowrap px-4 py-3 text-ellipsis">${chainId === 43111 ? "Hemi" : "Hemi Sepolia"}</td>
+  <td class="px-4 py-3">
+    <a class="font-mono text-neutral-500 hover:text-neutral-700" href="https://${chainId === 43111 ? "explorer" : "testnet.explorer"}.hemi.xyz/token/${address}" title="${address}">
```

Comment:
Really minor, but the `explorer` can be moved out of the ternary 😄 

```suggestion
    <a class="font-mono text-neutral-500 hover:text-neutral-700" href="https://${chainId === 43111 ? "" : "testnet."}explorer.hemi.xyz/token/${address}" title="${address}">
```